mscore/musescore.cpp

```diff
@@ -246,6 +252,9 @@ const std::list<const char*> MuseScore::_basicNoteInputMenuEntries {
             "",
             "pad-rest",
             "",
+            "pad-raise",
+            "pad-lower",
+            "",
```

Comment:
Good point, indeed those 3 buttons could get removed from the basic workspace/toolbar.
Your new buttons too shouldn't be in there either though, IMHO.